### PR TITLE
added Rest methods

### PR DESCRIPTION
**Pull request by Ivefan**

### CODE REVIEW

File: license-tldr/src/main/java/yasp/group/rest/LicenceResource.java [hunk 1]
```diff
@@ -10,44 +12,121 @@
 import java.util.List;
 
 @Singleton
-// WHY TF WAS LICENSES SPELLED WITH A C!?
-// I SPENT AT LEAST 6 HOURS ON THIS GARBAGE
-//@Path("/licences")
```
Comment (Hanicef) [inline license-tldr/src/main/java/yasp/group/rest/LicenceResource.java]: ( ͡° ͜ʖ ͡°)

File: license-tldr/src/main/java/yasp/group/rest/LicenceResource.java [hunk 2]
```diff
@@ -10,44 +12,121 @@
...
+	@Produces("application/json")
+	@Path("/summary/{summaryId}")
+	public Response getSummaryById(@PathParam("summaryId") int id) {
+		Summary result = service.getSummaryById(id);
+		return Response.ok(result).build();
+	}
+
+	@GET
+	@Produces("application/json")
+	@Path("/licsum/{LicSumId}")
```
Comment (Hanicef) [inline license-tldr/src/main/java/yasp/group/rest/LicenceResource.java:53]: "licsum"  ( ͡° ͜ʖ ͡°)

Comment (ArastoSahbaei) [reply license-tldr/src/main/java/yasp/group/rest/LicenceResource.java:53]: 👍

**Review by Hanicef — APPROVED** [review]
gud

**Pull request merged**